tests/tests/bitasset_tests.cpp

```diff
@@ -47,10 +47,21 @@ using namespace graphene::chain::test;
 
 BOOST_FIXTURE_TEST_SUITE( bitasset_tests, database_fixture )
 
-BOOST_AUTO_TEST_CASE( reset_backing_asset_on_witness_asset )
+BOOST_AUTO_TEST_CASE( reset_backing_asset_on_witness_asset)
```

Comment:
Coding style: the white space before `)` was intended.

Reply:
Done